content/projects/index.adoc

```diff
@@ -133,7 +133,7 @@ link:remoting[*Learn more*]
 
 == Jenkins Configuration as Code
 
-image:/images/logos/JCasC/JCasC.svg["Jenkins Configuration as Code", role=right, height=180]
+image:/images/logos/JCasC/JCasC.svg["Jenkins Configuration as Code", role=right, width=180]
```

Comment:
```suggestion
image:/images/logos/JCasC/JCasC.svg["Jenkins Configuration as Code", role=right, width=128]
```
A little bit smaller is better compared to the other images (in my opinion)

Reply:
Note: I would also apply this width of 128px to the Jenkins Operator image line 149 for a better uniformity.

Reply:
@Mahmoudgalalz PTAL at these suggestions.

Reply:
I like it, it makes the images are more same in the size
the last one K8s is not follow the same size 
so I will make it more smaller, 
do you think that should be done?

Reply:
@Mahmoudgalalz In my opinion yes I think both these modifications should be done.

Reply:
@lemeurherve done

Reply:
> This should not be merged as-is because the underlying issue is caused by #6329, which has been reverted in #6352

> This should not be merged as-is because the underlying issue is caused by #6329, which has been reverted in #6352

agreed, I did see that it was fixed when removing the property
but it will not change anything even if the problem has been fixed or not 
I see no reason for merging this but just the last commit is a good one to merge 

Thanks @NotMyFault